package.json

```diff
@@ -12,8 +12,13 @@
     "lint": "eslint src/* --ext .ts,.tsx",
     "test": "jest",
     "prepublishOnly": "npm run build",
-    "build": "rimraf dist/* && rollup -c --bundleConfigAsCjs"
+    "build": "rimraf dist/* && rollup -c --bundleConfigAsCjs",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json}\""
```

Comment:
If using prettierignore, why not simplify this to "prettier --write"?

Reply:
You are right, that way we only have a single source of truth for excluding files.